### DISCUSSION TITLE
Add a second flag for country representation

### DISCUSSION THREAD

**Discussion by OhCanada1867** [ppy/osu#30089] [Ideas]
This is from the perspective of someone who does not fully understand why your country flag can't be changed in the first place so please keep that in mind, anyone willing to clarify would be appreciated if that affects the suggestion:

I know I am not alone in feeling more connected to a country that isn't the one labeled on my profile, as an example, **[Ninerik](https://www.reddit.com/r/osugame/comments/15ans4v/ninerik_dual_citizens_opinion_on_flag_changes/?rdt=51096)** comes to mind. This proposal is not intended to replace or disregard the current country flag as you can change it after a period of time to your country of residence and for country based tournaments, this would still be valuable. All I am suggesting is a second "cosmetic" flag similar to how you can represent a country during OWC that would appear on your profile to be the country **you want to represent** not the country where you made your account.

Here are a couple design ideas I made in 10 minutes with crappy graphic design skills:

**Draft 1:**
![draft1](https://github.com/user-attachments/assets/eecd19a7-2095-4f44-bd3b-e01e27708de4)
_Original Flag Location_

**Draft 2:**
![draft2](https://github.com/user-attachments/assets/8d2cf1e5-4e44-49cd-b09d-51ae9c0e28df)
_Right Side of Tab_

**Draft 3:**
![draft3](https://github.com/user-attachments/assets/4d2e224a-4b4d-41b0-aca2-db25bf586f9a)
_Below in same tab as account creation date and playstyle_

My personal choice would be Draft 3 but that might detract from it too much or not be differentiate the two enough

As always, thank you for your consideration and keep up the good work dev team :)

**Comment by peppy** [comment]
nah not gonna happen. if this is a thing it should be at tournament level, not profile.
